code/script.js

```diff
@@ -29,23 +37,246 @@ const showMessage = (message, sender) => {
+            <button type="submit">Send</button>
+        </form>`
+
+
+  let form2 = document.getElementById('form2');
+  form2.addEventListener('submit', (event) => {
+    event.preventDefault()
+    confirmed(selectedTreatment, selectedDate, firstName)
+  })
+}
+
+
+
+const confirmed = (selectedTreatment, selectedDate, firstName) => {
+    //Transform selection strings to lowercase (for use within a sentence)
+    const selectedTreatmentValue = selectedTreatment.value;
+    const selectDateValue = selectedDate.value;
+    const lowerCaseTreatment = selectedTreatmentValue.toLowerCase();
+    const lowerCaseDate = selectDateValue.toLowerCase();
+
+  let userEmail = document.getElementById('userEmail');
+  userReply(userEmail.value);
+
+  botReply(`Thank you ${firstName}! You are welcome ${lowerCaseDate} for ${lowerCaseTreatment} with us. Have a wonderful day!`);
+
+  inputWrapper.innerHTML = `<p class="pressScissorText">Press the scissor to reload the page!</p>`;
+}
+
+const denied = (firstName) => {
+  userReply(`No`)
+  botReply(`What a shame. Maybe next time? Have a great day!`)
+
+  inputWrapper.innerHTML = `<p class="pressScissorText">Press the scissor to reload the page!</p>`;
+
+}
+
+
+const openingHours = (firstName) => {
+
+  userReply(`Opening hours`);
+  botReply(`Weekdays 10-22, 
+    Saturdays 10-18 & 
+    Sundays 11-15`)
+  botReply(`Hope to see you soon!`)
+
+      inputWrapper.innerHTML =
+      `<button id="backBtn" type="submit" value="Book appointment">Go back please</button>
+      `;
+
+      let backBtn = document.getElementById('backBtn');
+      backBtn.addEventListener('click', () => recognizeName(firstName))
+}
+
+
+const styleAdvice = (firstName) => {
+  userReply(`What should I do with my hair?`);
+
+  //console.log(`randomStyleAdvice.length`, randomStyleAdvice.length)
+  let index = getRandomNumber(0, randomStyleAdvice.length-1);
+  //console.log(`index`, index)
+
+  setTimeout(() => botReply(randomStyleAdvice[index]), 1500)
+
+  inputWrapper.innerHTML =
+      `<button id="againBtn" type="submit" value="Again">Again plz</button>
+      <button id="backBtn2" type="submit" value="Book appointment">Go back please</button>`;
+
+      let againBtn = document.getElementById('againBtn');
+      againBtn.addEventListener('click', () => newAdvice())
+
+      let backBtn2 = document.getElementById('backBtn2');
+      backBtn2.addEventListener('click', () => recognizeName(firstName))
+}
+
+//Give user new advice
+const newAdvice = () => {
+  userReply(`Not feeling it. Do you have another idea?`);
+
+  //console.log(`randomStyleAdvice.length`, randomStyleAdvice.length)
+  let index = getRandomNumber(0, randomStyleAdvice.length-1);
+  //console.log(`index`, index)
+
+  setTimeout(() => botReply(randomStyleAdvice[index]), 2000)
+}
+
+
+//Reloads the page
+const reloadSite = () => {
+    document.location.reload()
+}
+
+//Use for the randomizer
```

Comment:
It would be easier to follow the randomizer step by step if it where next to "let index" on row 230.